src/main/kotlin/com/jetbrains/snakecharm/stringLanguage/lang/SmkSLPythonRegExpInjector.kt

```diff
@@ -2,14 +2,23 @@ package com.jetbrains.snakecharm.stringLanguage.lang
 
 import com.intellij.lang.injection.MultiHostRegistrar
 import com.intellij.psi.PsiElement
+import com.intellij.psi.util.PsiTreeUtil
 import com.intellij.psi.util.elementType
 import com.jetbrains.python.codeInsight.PyInjectionUtil
 import com.jetbrains.python.codeInsight.PyInjectorBase
 import com.jetbrains.python.codeInsight.regexp.PythonRegexpLanguage
+import com.jetbrains.python.psi.PyArgumentList
+import com.jetbrains.python.psi.PyCallExpression
+import com.jetbrains.python.psi.PyReferenceExpression
 import com.jetbrains.python.psi.PyStringLiteralExpression
+import com.jetbrains.python.psi.impl.PyKeywordArgumentImpl
+import com.jetbrains.snakecharm.lang.SnakemakeLanguageDialect
+import com.jetbrains.snakecharm.lang.SnakemakeNames.WORKFLOW_WILDCARD_CONSTRAINTS_KEYWORD
+import com.jetbrains.snakecharm.lang.psi.SmkRuleOrCheckpointArgsSection
+import org.jetbrains.annotations.Nullable
 
 open class SmkSLPythonRegExpInjector : PyInjectorBase() {
-    override fun getInjectedLanguage(element: PsiElement) = PythonRegexpLanguage()
+    override fun getInjectedLanguage(element: PsiElement) = PythonRegexpLanguage.INSTANCE
```

Comment:
Have your ensured, that ` PythonRegexpLanguage` regexp dialect is ok here and fully supported? Better to check how it is implemented in snakemake sources. Maybe we need an independent regexp language close to `PythonRegexpLanguage` but with some features turned off/on

Reply:
See: https://github.com/JetBrains-Research/snakecharm/issues/182#issuecomment-672838662